config/crd/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+# This kustomization.yaml is not intended to be run by itself,
+# since it depends on service name and namespace that are out of this kustomize package.
+# It should be run by config/default
+resources:
+- bases/federation.openshift-service-mesh.io_federatedservices.yaml
```

Comment:
As we are using Helm, we are not adhering to `kustomize` structure. This folder does not exist as we put our manifests elsewhere. We will have to investigate how to align generation of those manifests.

Reply:
Created #139 to track this work